app/components/app-nav-primary.hbs

```diff
@@ -0,0 +1,42 @@
+{{#if this.show}}
```

Comment:
So, this makes is so the `<div>` with the `<nav>` in it isn't in the DOM unless `this.show` is true. Is that what we want here? Feels like it should be there and just moved off the side. That way, we can animate it in and (potentially) provide a better experience for screen readers. If not, then there's no real reason to have a specific class on the nav for being open vs closed.

Reply:
This is true... I'll adjust some css and classes to ditch any of my old animation code attempts and keep it simple for now, it's shown or not shown. The animation isn't working currently anyway :P great point